src/wizard/wizard.scss

```diff
@@ -1,57 +1,56 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
+@import '../text/index';
 
-@mixin iui-wizards {
+@mixin iui-wizard {
   @include iui-reset;
   @include iui-font-family;
 
-  > .iui-wizards-wrapper {
+  .iui-wizard-container {
+    @include iui-reset;
     display: flex;
     flex-direction: row;
     justify-content: space-around;
     align-items: flex-start;
     width: 100%;
     margin-bottom: $iui-baseline * 0.5;
+  }
 
-    > .iui-wizards-step {
-      align-items: center;
-      display: flex;
-      flex-direction: column-reverse;
-      font-size: $iui-font-size-small;
-      font-weight: $iui-font-weight-normal;
-      line-height: $iui-line-height;
-      position: relative;
-      width: 20%;
-
-      @include themed {
-        color: t(iui-color-background-positive);
-      }
+  .iui-wizard-step {
+    flex: 1 1 0;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    @include iui-text-small;
+    @include themed {
+      color: t(iui-color-background-positive);
+    }
 
-      &:hover {
-        text-decoration: none;
-      }
+    .iui-step-track {
+      display: flex;
+      align-items: center;
+      width: 100%;
 
-      > .iui-wizards-step-title {
-        text-align: center;
+      .iui-track {
+        height: $iui-xxs;
+        flex: 1 2 auto;
         @include themed {
-          color: t(iui-text-color-muted);
+          background-color: t(iui-color-background-5);
         }
       }
 
-      > .iui-wizards-step-circle {
-        align-items: center;
+      .iui-circle {
         border-radius: 50%;
-        display: flex;
-        height: $iui-l;
         width: $iui-l;
-        justify-content: center;
+        height: $iui-l;
```

Comment:
Is this a valid scenario? 😃 If yes, then we need to remove height in workflow and add `word-break: break-word`

![image](https://user-images.githubusercontent.com/36186912/128303322-4b7a81be-ead6-4606-9e84-7d5ff5b373a7.png)

Reply:
I think reverting `.iui-circle` back to flex fixed this:
![Screen Shot 2021-08-05 at 1 23 41 PM](https://user-images.githubusercontent.com/849817/128394059-a81a01f0-46ee-46c9-bd16-4e210621cc19.png)

Reply:
I was able to get this working.
![Screen Shot 2021-08-05 at 1 30 04 PM](https://user-images.githubusercontent.com/849817/128394776-1151b929-74c8-4c05-a2b5-f40e893f9e6e.png)